tests/brontology/relation_extraction/relation_types/test_reflexive_relation.py

```diff
@@ -0,0 +1,195 @@
+import spacy
+from brontology.relation_extraction.model import TokenRelation
+
+"""from brontology.relation_extraction.relation_types.reflexive_relation import (
+    extract_reflexive_relation,
+)"""
+from spacy.tokens import Token
+import en_core_web_trf
+from typing import Optional
+from spacy.tokens import Token
+from typing import Optional
+
+# Lade das Spacy Modell global
+nlp = spacy.load("en_core_web_trf")
+
+
```

Comment:
Musste die Funktion extract_reflexive_relation einfügen und die Sentence definieren.. Denke da können wir morgen nochmal drüber sprechen. Bin sonst immer in eine Fehlermeldung gelaufen.